en/developer_guide/api.rst

```diff
@@ -0,0 +1,120 @@
+API
+===
+
+API access
+----------
+
+The connection to the API is based on the OAuth2 protocol.
+The OAuth2 protocol provides a client a secure access to an API on behalf of an user.
+
+A client is represented by a key/secret pair.
+In order to use the API, the client must request a token.
+
+The token is a string that refers to the client and optionally to the user. It is used to authenticate the client
+when he makes a request to the API.
+
+
+Client
+------
+
+The API clients can be managed in the administration section of the back office.
+
+The creation of a client is done through the add form.
+In the form the key/secret pair is automatically generated by the server.
+
+Moreover, in this form the client can be specified like trusted.
+This parameter allows to authorize or not the client to use the API.
+
+Token
+-----
+
+Open Orchestra implements two ways to get a token. Both of those strategies are described in the OAuth2 protocol.
+
+Client credentials
+~~~~~~~~~~~~~~~~~~
+
+This strategy provides a token linked only to an API client.
+
+To obtain it, send the request:
+
+.. code-block::
+
+    /oauth/access_token?grant_type=client_credentials
+
+with HTTP header:
+
+.. code-block::
+
+    Authorization: Basic ENCODED_PAIR
+
+Where ``ENCODED_PAIR`` is the string ``key:secret`` encoded in base64.
+It's capital that the string to encode is "key:secret" (obviously key and secret depend on the client).
```

Comment:
do not say ``it's capital``. If you want to make an important note, you should put it between **